tests/format/angular/angular/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1147,7 +1147,8 @@ printWidth: 1
   (target)="
     a[
       b
-    ] = 1
```

Comment:
This is an extreme test with printWidth set to 1. I don't think it would be a problem in the real world.